### PR TITLE
Additional functionality requested from various people

### PR DESCRIPTION
**Pull request by lkastner**
- `ehrhart_polynomial(P::Polyhedron)`
- `issimple(P::Polyhedron)`
- fixes to docs

### CODE REVIEW

**Comment by thofma** [comment]
Would be good to have a version of the function, which takes the polynomial ring as an argument. Probably the first argument?

**Comment by fieker** [comment]
or with a parent= as the last argument?
Like in map_coefficients?

**Comment by lkastner** [comment]
Since this is more your area of expertise, I will implement the solution you agree on.

File: src/PolyhedralGeometry/Polyhedron/properties.jl [hunk 1]
```diff
@@ -736,7 +781,24 @@ julia> isbounded(P)
...
 ```
 """
-isbounded(P::Polyhedron) = pm_object(P).BOUNDED
+isbounded(P::Polyhedron) = pm_object(P).BOUNDED::Bool
+
+
+@doc Markdown.doc"""
+    issimple(P::Polyhedron)
+
+Check whether `P` is simple
```
Comment (fingolfin) [inline src/PolyhedralGeometry/Polyhedron/properties.jl]:
```suggestion
Check whether `P` is simple.
```

**Review by fingolfin — APPROVED** [review]
Minor typo, otherwise fine, thanks

File: src/PolyhedralGeometry/Polyhedron/properties.jl (hunk 1 above)
Comment (lkastner) [reply src/PolyhedralGeometry/Polyhedron/properties.jl]: Thanks.

**Comment by thofma** [comment]
Was this urgent? There was still an open question.

**Comment by lkastner** [comment]
Oh sorry, since no one replied and I implemented a version of the requested functionality I thought it was ok. I'll restore.

**Comment by lkastner** [comment]
I just had two people asking for `ehrhart_polynomial` for their next projects.

**Pull request merged**